pulp_ansible/app/models.py

```diff
@@ -527,9 +535,9 @@ def finalize_new_version(self, new_version):
         remove_duplicates(new_version)
         validate_repo_version(new_version)
 
-    @hook(BEFORE_UPDATE, when="remote", has_changed=True)
-    def _reset_repository_last_synced_metadata_time(self):
-        self.last_synced_metadata_time = None
```

Comment:
Why did you remove this hook? Is it not needed anymore?

Reply:
I think that was an accident. Let me re-add it.